backend/cfg/cfg.ml

```diff
@@ -284,19 +284,24 @@ let dump_op ppf = function
   | Floatop (Float32, op) ->
     Format.fprintf ppf "float32op %a" Printmach.floatop op
   | Csel _ -> Format.fprintf ppf "csel"
-  | Valueofint -> Format.fprintf ppf "valueofint"
-  | Intofvalue -> Format.fprintf ppf "intofvalue"
-  | Vectorcast Bits128 -> Format.fprintf ppf "vec128->vec128"
-  | Scalarcast Float32_as_float -> Format.fprintf ppf "float32 as float"
-  | Scalarcast (Float_of_int Float64) -> Format.fprintf ppf "int->float"
-  | Scalarcast (Float_to_int Float64) -> Format.fprintf ppf "float->int"
-  | Scalarcast (Float_of_int Float32) -> Format.fprintf ppf "int->float32"
-  | Scalarcast (Float_to_int Float32) -> Format.fprintf ppf "float32->int"
-  | Scalarcast Float_of_float32 -> Format.fprintf ppf "float32->float"
-  | Scalarcast Float_to_float32 -> Format.fprintf ppf "float->float32"
-  | Scalarcast (V128_to_scalar ty) ->
+  | Reinterpret_cast V128_of_v128 -> Format.fprintf ppf "vec128 as vec128"
```

Comment:
(Could be factored out with `Printcmm`
and `Printmach`.)

Reply:
Actually I'm not sure we should factor this out: in general it's better if IRs don't share types.

Reply:
I think they already do, the constructors
are in `Cmm` (`reinterpret_cast` and
`static_cast`).

Reply:
Factored out - maybe we should separate the cast types, but it was already like that